doc/tutorials/namespaces.md

```diff
@@ -6,25 +6,21 @@ The _key database_ of Elektra is _hierarchically structured_. This means that ke
 
 Let us add some keys to the database. To add a key we can use `kdb`, the _key database access tool_:
 
-```sh
+```shell
 kdb set <key> <value>
 ```
 
-Now add the key **/a** with the Value **Value 1** and the key **/b/c** with the Value **Value 2**:
+Now add the key **user:/tests/a** with the Value **Value 1** and the key **user:/tests/b/c** with the Value **Value 2**
 
 ```sh
-kdb set /a 'Value 1'
-#> Using name user:/a
-#> Create a new key user:/a with string "Value 1"
-kdb set /b/c 'Value 2'
-#> Using name user:/b/c
-#> Create a new key user:/b/c with string "Value 2"
+kdb set user:/tests/a 'Value 1'
+#> Create a new key user:/tests/a with string "Value 1"
+kdb set user:/tests/b/c 'Value 2'
+#> Create a new key user:/tests/b/c with string "Value 2"
 ```
 
-<img src="https://cdn.rawgit.com/ElektraInitiative/libelektra/master/doc/images/tutorial_namespaces_hierarchy.svg" alt="Hierarchical structure of key database" />
-
-Here you see the internal structure of the database after adding the keys **/a** and **/b/c**.
-For instance the key **/b/c** has the path **/** -> **b** -> **c**.
+Here you see the internal structure of the database after adding the keys **user:/tests/a** and **user:/tests/b/c**.
```

Comment:
> Here you see the internal structure of the database after adding the keys **user:/tests/a** and **user:/tests/b/c**.

This statement is not really suited anymore, because the image above which it refers to was removed. So please update the image or reword the sentence.